api/net/Areas/Admin/Controllers/TopicController.cs

```diff
@@ -54,7 +54,7 @@ public TopicController(ITopicService service)
     [SwaggerOperation(Tags = new[] { "Topic" })]
     public IActionResult FindAll()
     {
-        return new JsonResult(_service.FindAll().Select(ds => new TopicModel(ds)));
+        return new JsonResult(_service.FindAll().Where(t => t.IsEnabled).Select(ds => new TopicModel(ds)));
```

Comment:
Make the filter on the UI not here.  We want to returned disabled topics so that if content has that value set it will still show up.

Reply:
Updated, thanks